react-native/setup/setup-stripe-dev-standard.mo.md

```diff
@@ -0,0 +1,88 @@
+### [MO] Setup Stripe in app payment form with tispi-stripe (~2 hours)
+
+#### Owner [Guillaume Renouvin](https://github.com/GuillaumeRenouvin)
+
+#### Prerequisites
+- Have a Stripe account and its publishable/secret key duet
+
+#### Context
+On a project, I had to let my user pay within my React Native app. My client wanted to use Stripe services.
+
+First I tried to develop a browser solution with [stripe element](https://stripe.com/docs/stripe-js/elements/quickstart). I had two problems:
+- It was really time consuming to develop a web page for the payment;
+- It made the user navigate back and forth from the app to his web browser, resulting in a really poor user experience.
+
+I then found out [tipsi-stripe](https://github.com/tipsi/tipsi-stripe) which is a React Native library that wraps the Stripe native SDK for iOS and Android. It only took me 2 hours to set it up and granted a far better user experience, satisfying our client ;).
+
+#### However your client should be aware of
+- [tipsi-stripe](https://github.com/tipsi/tipsi-stripe) only supports english;
+- [tipsi-stripe](https://github.com/tipsi/tipsi-stripe) has not been audited by Stripe yet. The PCI DSS (Payment Card Industry Data Security Standard) regulation committee recently forced Stripe to stop accepting credit card data from direct calls to their API. Therefore, they enforced the usage of "Stripe-controlled" libraries like Elements and their own SDKs.  
+**It means, Stripe could actually ask you to stop using tipsi in order to fit new PCI DSS requirements anytime.**  
+So far, tipsi-stripe uses direct calls to Stripe SDKs and therefore, we did not receive any warning from them, still you should keep it in mind.
+
+However, even if Stripe asked you to stop using tipsi, the time loss would be light as it only takes around 2 hours to install the lib. Setting up browser payment would take more than a day (Ledon, Neo-nomade, ...) and offers a poorer UX.
+
+
+#### Steps (~1 hour)
+- Install tipsi-stripe package following his [installation documentation](https://github.com/tipsi/tipsi-stripe#installation)
```

Comment:
How to check it works after install:
- break point in ios in the code
- break point in android in the code
- console.log(NativeModules) where NativeModules is a react native import